### PR TITLE
Partner feature added.

### PR DESCRIPTION
**Pull request by Mschuddings**

### CODE REVIEW

File: lib/features/custom/erasmus_30_year_anniversary_partner/erasmus_30_year_anniversary_partner.views_default.inc [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  $handler->display->display_options['filters']['type']['field'] = 'type';
+  $handler->display->display_options['filters']['type']['value'] = array(
+    'erasmus_30_year_anniversary_part' => 'erasmus_30_year_anniversary_part',
+  );
+  /* Filter criterion: Content: Country (field_country_na_events) (translated) */
+  $handler->display->display_options['filters']['field_country_na_events_tid_i18n']['id'] = 'field_country_na_events_tid_i18n';
+  $handler->display->display_options['filters']['field_country_na_events_tid_i18n']['table'] = 'field_data_field_country_na_events';
+  $handler->display->display_options['filters']['field_country_na_events_tid_i18n']['field'] = 'field_country_na_events_tid_i18n';
+  $handler->display->display_options['filters']['field_country_na_events_tid_i18n']['value'] = array(
+    152 => '152',
```
Comment (iulianweb) [inline lib/features/custom/erasmus_30_year_anniversary_partner/erasmus_30_year_anniversary_partner.views_default.inc]: Hardcoded tids should not be here, this is dangerous...

File: lib/features/custom/erasmus_30_year_anniversary_partner/erasmus_30_year_anniversary_partner.views_default.inc [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    143 => '143',
+    192 => '192',
+  );
+  $handler->display->display_options['filters']['field_country_na_events_tid_i18n']['exposed'] = TRUE;
+  $handler->display->display_options['filters']['field_country_na_events_tid_i18n']['expose']['operator_id'] = 'field_country_na_events_tid_i18n_op';
+  $handler->display->display_options['filters']['field_country_na_events_tid_i18n']['expose']['label'] = 'Country';
+  $handler->display->display_options['filters']['field_country_na_events_tid_i18n']['expose']['operator'] = 'field_country_na_events_tid_i18n_op';
+  $handler->display->display_options['filters']['field_country_na_events_tid_i18n']['expose']['identifier'] = 'country';
+  $handler->display->display_options['filters']['field_country_na_events_tid_i18n']['expose']['remember_roles'] = array(
+    2 => '2',
```
Comment (iulianweb) [inline lib/features/custom/erasmus_30_year_anniversary_partner/erasmus_30_year_anniversary_partner.views_default.inc]: Same as above.

File: lib/features/custom/erasmus_30_year_anniversary_partner/erasmus_30_year_anniversary_partner.info [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+features[variable][] = node_options_erasmus_30_year_anniversary_part
+features[variable][] = node_preview_erasmus_30_year_anniversary_part
+features[variable][] = node_submitted_erasmus_30_year_anniversary_part
+features[views_view][] = erasmus_30_year_anniversary_partners
+features_exclude[dependencies][ctools] = ctools
+features_exclude[dependencies][features] = features
+features_exclude[dependencies][filefield_paths] = filefield_paths
+features_exclude[dependencies][image] = image
+features_exclude[dependencies][link] = link
+project path = sites/all/modules/features/custom
```
Comment (iulianweb) [inline lib/features/custom/erasmus_30_year_anniversary_partner/erasmus_30_year_anniversary_partner.info]: Please remove project path.

File: lib/features/custom/erasmus_30_year_anniversary_partner/erasmus_30_year_anniversary_partner.views_default.inc (hunk 1 above)
Comment (Mschuddings) [inline lib/features/custom/erasmus_30_year_anniversary_partner/erasmus_30_year_anniversary_partner.views_default.inc]: Dear,

Could you please elaborate on the fact why they are dangerous.
This taxonomy and user role are already existing in the multisite platform so there cannot be an issue with matching with the wrong terms.

The second issue is that these files are generated by features.

The only solution I see is that we have to manually edit the files after re-exporting the views and create code that puts these "tids" in a variable.

Kind regards,
Michael

File: lib/features/custom/erasmus_30_year_anniversary_partner/erasmus_30_year_anniversary_partner.views_default.inc (hunk 2 above)
Comment (Mschuddings) [inline lib/features/custom/erasmus_30_year_anniversary_partner/erasmus_30_year_anniversary_partner.views_default.inc]: See above please.

File: lib/features/custom/erasmus_30_year_anniversary_partner/erasmus_30_year_anniversary_partner.info (hunk 3 above)
Comment (Mschuddings) [inline lib/features/custom/erasmus_30_year_anniversary_partner/erasmus_30_year_anniversary_partner.info]: Done

File: lib/features/custom/erasmus_30_year_anniversary_partner/erasmus_30_year_anniversary_partner.views_default.inc (hunk 1 above)
Comment (tatarbj) [inline lib/features/custom/erasmus_30_year_anniversary_partner/erasmus_30_year_anniversary_partner.views_default.inc]: Hi @Mschuddings,
actually all these hardcoded tids, uids, nids, etc should be avoided, especially when you make things on your local environment and export them to features. You can easily imagine how big problem could be generated if you create test contents on the environment where you work and accidentially the id doesn't match with the real ones, that's the reason why we always recomment this best practice where it's just possible to do not include hardcoded things here. As i see this issue with the one below could be accepted, or just removed from the exported views and make it manually happen on site or use a bit advanced way to make it via hook_views_query_alter(), as you wish :)
Cheers,
Balazs.

**Pull request merged**